project.clj

```diff
@@ -56,7 +56,8 @@
                       :dependencies [[org.clojure/clojure "1.8.0-master-SNAPSHOT"]]}
 
              :test-clj {:test-paths ["test/clj"]
-                        :java-source-paths ["test/java"]}
+                        :java-source-paths ["test/java"]
+                        :resource-paths ["test/resources"]}
```

Comment:
OK, just submitted a PR for this, will rebase this branch off master if it gets merged in